### ISSUE TITLE
items compoundtag is broken in 1.0

### ISSUE THREAD

**Issue by Creeperface01** [Nukkit/Nukkit#1395]
mcpe probably sends different nbt so its not equal with real item nbt, so player can not do anything with these items

**Comment by Creeperface01** [comment]
*coompound tag sent from client:

TAG_Compound: CompoundTag  (2 entries)
{
   TAG_List("ench"): ListTag ench [1 entries of type TAG_Compound]
   {
      TAG_Compound: CompoundTag  (2 entries)
      {
         TAG_Short("lvl"): 1
         TAG_Short("id"): 15
      }
   }
   TAG_Compound("display"): CompoundTag display (1 entries)
   {
      TAG_String("Name"): StringTag Name (data: žrž6SoulBound)
   }
}


* real item nbt

TAG_Compound("tag"): CompoundTag tag (2 entries)
{
   TAG_List("ench"): ListTag ench [1 entries of type TAG_Compound]
   {
      TAG_Compound: CompoundTag  (2 entries)
      {
         TAG_Short("lvl"): 1
         TAG_Short("id"): 15
      }
   }
   TAG_Compound("display"): CompoundTag display (1 entries)
   {
      TAG_String("Name"): StringTag Name (data: žrž6SoulBound)
   }
}

**Comment by Creeperface01** [comment]
but it begin after few minutes by using this item

**Comment by MrPowerGamerBR** [comment]
@Creeperface01 I think I'm blind, but what is the difference between the NBT tag from the client vs the real item nbt?

**Comment by Creeperface01** [comment]
first line

**Comment by MrPowerGamerBR** [comment]
@Creeperface01 oh, now that I seen it.

**Comment by Creeperface01** [comment]
idk how to solve it

**Comment by Creeperface01** [comment]
and if its MCPE or nukkit bug

**Comment by MrPowerGamerBR** [comment]
@Creeperface01 maybe a fix would be not sending the "tag" compound name

**Comment by MrPowerGamerBR** [comment]
@Creeperface01 actually the fix would be ignoring the "tag" name when Nukkit processes an action

**Comment by Creeperface01** [comment]
yes, but this bug appears only with enchantments probably, if there is display tag only, it works well

**Comment by Goodstuff4UonYT** [comment]
@Creeperface01 is this still a bug, or could it be closed?

**Comment by Creeperface01** [comment]
Still
